trymito.io/pages/industries/financial-services.tsx

```diff
@@ -0,0 +1,227 @@
+import Head from 'next/head';
+import Image from 'next/image';
+import Header from '../../components/Header/Header';
+
+import titleStyles from '../../styles/Title.module.css';
+import pageStyles from '../../styles/Page.module.css';
+import trifoldStyles from '../../styles/Trifold.module.css'
+import securityStyles from '../../styles/Security.module.css'
+import textImageSplitStyles from '../../styles/TextImageSplit.module.css'
+import ComputerIcon from '../../public/icon-squares/ComputerIcon.svg';
+import ReconIcon from '../../public/icon-squares/ReconIcon.svg';
+import ExcelIcon from '../../public/icon-squares/ExcelIcon.svg';
+import ReportIcon from '../../public/icon-squares/ReportIcon.svg';
+
+import { classNames } from '../../utils/classNames';
+import CTAButtons from '../../components/CTAButtons/CTAButtons';
+import DownloadCTACard from '../../components/CTACards/DownloadCTACard';
+import Footer from '../../components/Footer/Footer';
+
+const Security: NextPage = () => {
+
+    return (
+        <>
+            <Head>
+                <title>Mito | Security </title>
+                <meta name="viewport" content="width=device-width, initial-scale=1" />
+                <meta name="description" content="Best Python Package for Finance, Insurance, and Fintech | Mito" />
+            </Head>
+            
+            <Header/>
+        
+            <div className={pageStyles.container}>
+
+                <main className={pageStyles.main}>
+                    <section className={classNames(titleStyles.title_card, titleStyles.grid_card)}>
+                        <h1 className={titleStyles.title}>
+                            Finance professionals save themselves hours per week with Python.
+                        </h1>
+                        <p className={titleStyles.description}>
+                            The world&apos;s largest financial institutions are already using Mito to automate their spreadsheets.
+                        </p>
+                        <CTAButtons variant={'download'} align='center'/>
+                    </section>
+
+                    <section>
+                        <div className={classNames(pageStyles.subsection, 'center')}>
+                            <h2>
+                                Mito simplifies Python for Finance
+                            </h2>
+                        </div>
+                        <div className={classNames(pageStyles.subsection, trifoldStyles.container)}>
+                            <div className={classNames(pageStyles.subsection_column)}>
+                                <div className={classNames(trifoldStyles.image_container, 'margin-top-8rem-mobile-only')}>
+                                    <Image src={'/mito_generated_code.png'} alt='Large Data' width={1000} height={500}/>
+                                </div>
+                                <h3>
+                                    A software solution for Adopting Python and Pandas
```

Comment:
capitalization is strange

Reply:
Also, this is a bit abstract. Can we make it more concrete like "Make Python useful, immediately." or "Reduce time to learning python by weeks" or something

Reply:
Given by SEO team